packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -126,6 +127,14 @@ class ButtonBar extends StatelessWidget {
   /// If that is null, it will default [ButtonBarLayoutBehavior.padded].
   final ButtonBarLayoutBehavior layoutBehavior;
 
+  /// Defines the vertical direction of a [ButtonBar]'s children if it
+  /// overflows.
+  ///
+  /// If null then it will use the surrounding
+  /// [ButtonBarTheme.verticalDirection]. If that is null, it will
```

Comment:
This property has a specific meaning; maybe give it a more specific name. Like buttonBarOverflowDirection?